Configurations/10-main.conf

```diff
@@ -1687,13 +1687,32 @@ my %targets = (
 # CROSS_TOP=`xcode-select --print-path`/Platforms/iPhoneOS.platform/Developer
 # CROSS_SDK=iPhoneOS.sdk
 #
+    "ios-common" => {
+        inherit_from     => [ "BASE_unix" ],
+        template         => 1,
+        cc               => "cc",
+        cflags           => combine(picker(default => "",
+                                           debug   => "-g -O0",
+                                           release => "-O3"),
+                                   threads("-D_REENTRANT")),
+        sys_id           => "MACOSX",
+        plib_lflags      => "-Wl,-search_paths_first",
+        bn_ops           => "BN_LLONG RC4_CHAR",
+        thread_scheme    => "pthreads",
+        perlasm_scheme   => "osx32",
+        dso_scheme       => "dlfcn",
+        ranlib           => "ranlib -c",
+        shared_target    => "ios-shared",
```

Comment:
In other words this is the only field that is overridden. Note that it's possible to chain templates, so this could as well be
```
"ios-common" => {
    inherit_from => [ "darwin-common" ],
    template => 1,
    shared_target => "ios_shared",
}
```
However! It's also possible to override it in each target. And since there are only 3 of them, I for one would rather do that.  Since it's really no biggie. This is of course provided that we establish that it doesn't hurt normal builds.